packages/grafana-ui/src/components/MatchersUI/utils.ts

```diff
@@ -79,9 +79,20 @@ export function useSelectOptions(
       if (!found && name === currentName) {
         found = true;
       }
+      const field = displayNames.fields.get(name);
+      let previewValues = '';
+      if (field?.values.length && field.values.length > 2) {
+        previewValues += field.values.get(0);
+        previewValues += `, ${field.values.get(1)}`;
+        previewValues += `, ${field.values.get(2)}`;
+        if (field.values.length > 3) {
+          previewValues += '...';
+        }
+      }
       options.push({
         value: name,
         label: name,
+        description: field ? (previewValues.length > 0 ? `${field.type}: ${previewValues}` : field.type) : undefined,
```

Comment:
maybe:
```
"Type: string, Values: 1,2,3..."
```

Alternatively the "label" could be: `${name} (${field.type})` and description the first few values :man_shrugging:

Reply:
Isn't type indication sufficient? Why we want to display values preview as well?

Reply:
for sure type indication -- but showing the first few values is nice for pickers where you can not easily switch to the table view and correlate which values are in which field.  `SELECT * from table LIMIT 10` etc

I'm not sure it should be default or an option.  Lets see what it looks like in practice and evaluate.

Reply:
Added the first suggestion to start with and attaching a screenshot of the label suggestion as well.

With first suggestion:
<img width="173" alt="Screen Shot 2021-09-01 at 9 51 58 AM" src="https://user-images.githubusercontent.com/42276368/131712912-94cc35c8-5433-46f9-ab23-b5e2d10d2c87.png">


With the label:
<img width="182" alt="Screen Shot 2021-09-01 at 9 55 24 AM" src="https://user-images.githubusercontent.com/42276368/131712890-0b3c7df6-ff96-4bc4-80ba-145c3e374f3e.png">